receiver/jaegerreceiver/trace_receiver.modified.go

```diff
@@ -205,19 +199,11 @@ func (h *agentHandler) EmitBatch(ctx context.Context, batch *jaeger.Batch) error
 	return err
 }
 
-func (jr *jReceiver) GetSamplingStrategy(ctx context.Context, serviceName string) (*sampling.SamplingStrategyResponse, error) {
-	return jr.agentSamplingManager.GetSamplingStrategy(ctx, serviceName)
+func (notImplementedConfigManager) GetSamplingStrategy(ctx context.Context, serviceName string) (*sampling.SamplingStrategyResponse, error) {
```

Comment:
line in trace_receiver.go_original move to above `type agentHandler...` so that it's easier to diff the original and modified.